### ISSUE TITLE
date: support relative `--date`

### ISSUE THREAD

**Issue by PedroHLC** [uutils/coreutils#3463]
A simple example of inputs/outputs of what GNU's Coreutils is capable of is visible at https://www.cyberciti.biz/tips/linux-unix-get-yesterdays-tomorrows-date.html

This issue was suggested by @tertsdiepraam in https://github.com/uutils/coreutils/issues/2685#issuecomment-1111308354 (which might be interesting to solve in the same PR).

**Comment by Ecordonnier** [comment]
I think this was solved by https://github.com/uutils/coreutils/pull/4790

**Comment by cakebaker** [comment]
Yes, #4790 implements most of the functionality, though there is still some functionality missing (for example, "next week" and "last week" are not recognized yet).

**Comment by jfinkels** [comment]
There is now some support for specifying a relative time. For example:
```
$ TZ=UTC0 ./target/debug/date
Fri Feb 21 23:04:05 UTC 2025
$ TZ=UTC0 ./target/debug/date --date=tomorrow
Sat Feb 22 23:04:10 UTC 2025
```
The code is in the `parse_datetime` library.

I suggest we close this issue as "done enough" and open new ones for specific bugs.
